tests/server/TestConfig.js

```diff
@@ -1,69 +1,6 @@
 import { assert } from 'chai';
-import config from 'config';
 import requireUncached from 'require-uncached';
 
-import { getClientConfig } from 'core/utils';
-
-const appsList = config.get('validAppNames');
-
-
-// eslint-disable-next-line no-restricted-syntax
-for (const appName of appsList) {
-  describe(`Config for ${appName}`, () => {
-    let oldNodeEnv;
-
-    before(() => {
-      oldNodeEnv = process.env.NODE_APP_INSTANCE;
-    });
-
-    beforeEach(() => {
-      process.env.NODE_APP_INSTANCE = appName;
-      process.env.NODE_ENV = 'production';
```

Comment:
These tests are tricky because they depend on the production config which is overidable with a [local config](https://github.com/mozilla/addons-frontend#building-and-running-services). They are creating conflicts when `config/local-production-amo.js` is present. On closer inspection, they are just checking that the configs are sane. Do we really need tests for that?